### DISCUSSION TITLE
RESP3 support

### DISCUSSION THREAD

**Discussion by jakule** [redis/go-redis#1999] [Q&A]
Do you know when the support for Redis 6 RESP3 protocol will be released? I found this PR https://github.com/go-redis/redis/pull/1739 which was merted into v9 branch a few months ago, but the latest release is still v8.

**Comment by vmihailenco** [comment]
When https://github.com/go-redis/redis/issues?q=is%3Aissue+is%3Aopen+label%3Av9 are resolved.
